tests/core/test_audit.py

```diff
@@ -20,7 +20,7 @@
 def model() -> Model:
     return create_sql_model(
         "db.test_model",
-        parse_one("SELECT a, b, ds"),
+        parse_one("SELECT a, b, ds::varchar"),
```

Comment:
Every test in this class was based around the `time_column` being treated as a string, so I made this explicit instead of updating all the assertions